plugins/pencilblue/controllers/actions/salesforce/login_callback.js

```diff
@@ -57,7 +57,9 @@ module.exports = function LoginSalesforceCallbackControllerModule(pb) {
                 object_type: 'jobseeker_profile'
             };
             const hasJobSeekerProfile = await siteQueryService.loadByValuesAsync(query, 'jobseeker_profile');
-            if (state && state.redirectURL) {
+            if (state && state.highPriorityToRegister && !hasJobSeekerProfile) {
```

Comment:
Where or who defines hasJobSeekerProfile?

Reply:
I'm using the same check as before, I just check if there's some jobseeker_profile related with the external user id. You could open the complete file and you are going to be able to see the query.